Simplenote/src/androidTest/java/com/automattic/simplenote/utils/ChecklistsTest.java

```diff
@@ -23,9 +23,9 @@
 public class ChecklistsTest {
     @Test
     public void testRegexMatching() {
```

Comment:
What do you think about leaving the `testRegexMatching` unmodified and creating a new method for testing regular expression matching for nested checkboxes?  That way we will know the feature before these changes still works.

Reply:
Good idea, did that in https://github.com/Automattic/simplenote-android/pull/622/commits/56fd38ef5c555790790893e3ccdc7bee7b23b723